res/nvimrc

```diff
@@ -77,7 +99,8 @@ anoremenu .900 &Window.Toggle\ Full\ Screen :call MacToggleFullScreen()<CR>
 " Assign Cmd-key shortcuts
 call MacMenu("File.New Window", "T-n")
 call MacMenu("File.Save", "T-s")
-call MacMenu("File.Open...", "T-o")
+call MacMenu("File.Save As", "T-S-s")
+call MacMenu("File.Open", "T-o")
```

Comment:
@bambu actually, the ellipsis character is the correct one to use in menus, as demonstrated by Appel’s own apps. Try changing system wide shortcuts for such a menu item in System Preferences to see for yourself :).